### ISSUE TITLE
Have you got this working somehow?

### ISSUE THREAD

**Issue by xhevahir** [spdegabrielle/DivaScheme#1]
As I remember, DivaScheme stopped working with the change to Racket(v5.0). Have you got it working? I tried 'racket install.rkt' on this repo it reported a successful installation but DivaScheme doesn't show up anywhere in the interface. When I tried installing with the package manager I got this message on startup:
class*: cannot override or augment final method
  method name: set-surrogate
  class name: diva-link:text-mixin
  context...:
   /usr/share/racket/collects/racket/private/class-internal.rkt:4680:0: obj-error
   /usr/share/racket/collects/racket/private/class-internal.rkt:2630:32
   /usr/share/racket/collects/racket/private/map.rkt:267:2: gen-for-each
   /usr/share/racket/collects/racket/private/class-internal.rkt:2109:0: compose-class
   /home/wade/repos/DivaScheme/tool.rkt:74:8: diva-interactions-text-mixin
   /usr/share/racket/pkgs/drracket/drracket/private/get-extend.rkt:26:2
   /usr/share/racket/pkgs/drracket/drracket/private/get-extend.rkt:69:2: get-built
   .../racket/unit.rkt:1010:20
   body of "/usr/share/racket/pkgs/drracket/drracket/tool-lib.rkt"
   body of "/usr/share/racket/pkgs/drracket/drracket/private/drracket-normal.rkt"
   body of "/usr/share/racket/pkgs/drracket/drracket/drracket.rkt"

**Comment by gmarceau** [comment]
Hi xhevahir,

Amazing that you remember! DivaScheme has been sitting there all these
years without a maintenance unfortunately.

Which is too bad. It is a neat project I always been quite fond of. I would
like to see something happen of it.

I am adding Danny Too in cc since he also worked hard on it back in the day.

Best,

**Comment by xhevahir** [comment]
Hello, Guillaume
DivaScheme was indeed a nice project; I've hardly used DrRacket since it stopped working. Unfortunately I don't have a lot of expertise--for example, the object-oriented stuff I'm finding in the error messages is well over my head.

I looked at the repo and noticed that, although the .ss file extensions have been changed to .rkt, the strings referring to those files had not. Yet after I'd made those changes, installing the package gave me the same error.
